.env.example

```diff
@@ -24,6 +24,10 @@ WP_DEBUG_DISPLAY=false
 # Prevents log being filled with deprecation notices raised by BuddyBoss
 EXCLUDED_ERROR_LEVELS="E_DEPRECATED,E_USER_DEPRECATED"
 
+# Airtable API settings
+AIRTABLE_API_KEY=
+AIRTABLE_REPORTS_BASE=appxMRW8ERvV5bZuw
```

Comment:
Is this an example value?

Reply:
No, it's the actual identifier. Do you not want it included?

Reply:
Yeah, it shouldn't be included in the example file